packages/jaeger-ui/src/components/TraceDiff/TraceDiffGraph/drawNode.js

```diff
@@ -57,6 +59,9 @@ class DiffNode extends React.PureComponent<Props> {
             </td>
             <td className={`DiffNode--labelCell ${className}`}>
               <strong>{service}</strong>
+              <span className="DiffNode--copyIcon">
```

Comment:
You migth be able to skip having a wrapping node if you add a [`className` prop](https://github.com/jaegertracing/jaeger-ui/blob/046de99a5d384cf62f29eb27db1f0c49351c55c9/packages/jaeger-ui/src/components/common/NewWindowIcon.js#L27-L28) to the `CopyIcon` component?

Reply:
that is cleaner, added